### DISCUSSION TITLE
v2 changelog

### DISCUSSION THREAD

**Discussion by jakst** [mskelton/yarn-plugin-outdated#21] [General]
Is there a changelog for v2? Would be nice to know what changed 🙂

**Comment by mskelton** [comment]
The only change in V2 was the upgrade to Yarn 3 🙂   Since it wasn't backwards compatible, it required the major version bump.  Wish it was a more interesting release, but alas it isn't!

**Reply by jakst** [reply]
Good to know! 👍 Although, the previous version worked fine for me on yarn v3
